examples/with-redux-observable/README.md

```diff
@@ -29,24 +29,56 @@ npm run dev
 
 
 ### The idea behind the example
-Example is a page that renders information about Star-Wars characters. It fetches new character  
-every 3 seconds having the initial character fetched on a server.
+
+This example is a page that renders information about Star-Wars characters. It
+fetches new character every 3 seconds having the initial character fetched on
+a server.
 
 Example also uses `redux-logger` to log every action.
 
 ![demo page](demo.png)
 
-The main problem with integrating Redux, Redux-Observable and Next.js is probably making initial requests
-on a server. That's because it's not possible to wait until epics are resolved in `getInitialProps` hook. 
+The main problem with integrating Redux, Redux-Observable and Next.js is
+probably making initial requests on a server. That's because, the
+`getInitialProps` hook runs on the server-side before epics have been made available by just dispatching actions.
+
+However, we can access and execute epics directly. In order to do so, we need to
+pass them an Observable of an action and they will return an Observable:
+
+```js
+static async getInitialProps({ store, isServer }) {
+  const resultAction = await rootEpic(
+    of(actions.fetchCharacter(isServer)),
+    store
+  ).toPromise(); // we need to convert Observable to Promise
+  store.dispatch(resultAction)};
+```
+
+Note: we are not using `AjaxObservable` from the `rxjs` library; as of rxjs
+v5.5.6, it will not work on both the server- and client-side. Instead we call
+the default export from
+[universal-rx-request](https://www.npmjs.com/package/universal-rx-request) (as
+`ajax`).
+
+We transform the Observable we get from `ajax` into a Promise in order to await
+its resolution. That resolution should be a action (since the epic returns
+Observables of actions). We immediately dispatch that action to the store. 
```

Comment:
I'm not sure if/where we should mention this but in order to use `toPromise`, the stream must complete. While the epic here is written to complete because the source actions stream will complete, it is possible to write non-terminating epics (example: autosave that emits a `SAVE` action on an interval).

As with pretty much all issues while working with redux-observable, they end up being about fundamentals of Rx. I can't tell if it's best to bring up the epic termination bits here or not. ¯\\\_(ツ)\_/¯ People get tripped up with Rx (myself included).